### PR TITLE
 update FAQ and fix some link errors

### PR DESCRIPTION
**Pull request by kuailehaibin**
> + update FAQ in English and Chinese;
> + fix [Bug-Raspbian](https://github.com/baidu/openedge/issues/121);
> + fix some link errors.

### CODE REVIEW

File: doc/zh-cn/FAQ.md [hunk 1]
```diff
@@ -1,68 +1,90 @@
...
+设置环境变量 `DOCKER_API_VERSION=1.38`
 
 例，CentOs 下启动命令：
 
 ```shell
 export DOCKER_API_VERSION=1.38
 ```
 
-**问题7**: BIE如何接入NB-IoT?
+**问题 7**: BIE如何接入NB-IoT?
```
Comment (ludanfeng) [inline doc/zh-cn/FAQ.md]: OpenEdge如何使用NB-IoT连接百度云管理套件或者物接入

File: doc/us-en/FAQ.md [hunk 2]
```diff
@@ -64,4 +65,25 @@ export DOCKER_API_VERSION=1.38
...
\ No newline at end of file
+**Suggested Solution**: NB-IoT is a network standard similar to 2/3/4G with low bandwidth and low power consumption. NB-IoT supports TCP-based MQTT protocol, so you can use NB-IoT card to connect to Baidu Cloud IotHub, deploy OpenEdge application and communicate with BIE Cloud Management Suite. However, among the three major operators in China, Telecom have imposed whitelist restrictions on their NB cards, and only allow to connect to Telecom Cloud Service IP. Therefore, only Mobile NB cards and Unicom NB cards can be used to connect to Baidu Cloud Service.
+
+**Question 8**：According to OpenEdge's design, how many MQTT clients can be connected to One OpenEdge?
```
[the remark this answers is not in the document]
Comment (kuailehaibin) [reply doc/us-en/FAQ.md]: ok

File: doc/zh-cn/FAQ.md [hunk 3]
```diff
@@ -1,68 +1,90 @@
...
+**参考方案**：接入 MQTT 客户端的数量依赖边缘设备的硬件配置和性能。
+
+**问题 9**：在购买的 NXP LS1046 ARDB 的盒子上部署了 OpenEdge（容器模式），但是启动时候出现 `{"errorDetail": {"message":"no matching manifest for linux/arm64 in the manifest list entries"}, "error":"no matching manifest for linux/arm64 in the manifest list entries"}` 错误信息。
+
+**参考方案**：出现上述问题是由于 OpenEdge 加载执行时候，会根据系统 CPU 类型拉取对应平台镜像启动，而目前 OpenEdge 容器模式暂未支持 Linux/arm64 平台镜像，后续发布版本会加入支持。
+
+**问题 10**：在应用 OpenEdge Hub 模块测试 MQTT 客户端连接时，怎样使用正确的用户名、密码（Hub 模块配置文件存储密码为原明文密码的 SHA256 值）进行连接？
+
+**参考方案**：提供两种方案：（1）在云端管理 Console 创建边缘核心时，显示的窗口中会展示连接使用的用户名和密码（明文，下发后按照 SHA256 值存储），因此，在创建核心时记录即可（**推荐**）；（2）如果启动时候也应用了其他模块，如 Remote 远程模块，Function 函数计算模块，其相应的配置文件中会存储与 Hub 模块连接时应用的用户名和密码（其他模块与 Hub 模块连接，作为 Hub 的客户端），直接获取即可。
+
```
[the remark this answers is not in the document]
Comment (kuailehaibin) [reply doc/zh-cn/FAQ.md]: 好的

File: doc/zh-cn/FAQ.md (hunk 1 above)
Comment (kuailehaibin) [reply doc/zh-cn/FAQ.md]: 好的

**Pull request merged**